src/foam/u2/view/RadioView.js

```diff
@@ -87,7 +87,7 @@ foam.CLASS({
     function renderContent() {
       var self = this;
 
-      this.add(function(choices) {
+      this.add(this.slot(function(choices) {
```

Comment:
The problem is that this should be without the 'this.slot(' but then on line 96 it should be 'this' instead of 'self' and on line 91 it shouldn't 'return' and should call forEach() instead of map(). Let me know if you need and explanation of why.

Reply:
Makes sense for the most part - I was just being lazy with the fix.
After further investigation the issue is coming from element function rendering:
<img width="794" alt="Screenshot 2025-04-15 at 8 13 32 am" src="https://github.com/user-attachments/assets/9ce4ef6f-9ca2-4747-a583-a6fd80acdc35" />

Just noting - for now - will see what I can do to address.